enterprise/backend/src/metabase_enterprise/metabot_v3/tools/api.clj

```diff
@@ -496,6 +509,17 @@
                          [:result_columns ::columns]]]]
    [:map [:output :string]]])
 
+(mr/def ::get-report-details-arguments
+  [:and
+   [:map
+    [:report_id                                         :int]
+    [:with_fields       {:optional true, :default true} :boolean]
+    [:with_field_values {:optional true, :default true} :boolean]]
```

Comment:
I kept the names `with_fields` and `with_field_values`, although the property they influence is called `result_columns`. I can change this if this is difficult or misleading on the caller side.